toplevel/toploop.mli

```diff
@@ -15,6 +15,11 @@
 
 open Format
 
+(* type of toplevel usages *)
+type input_type =
```

Comment:
I would call this just `input` (of course it's a type). Also, the comment above would need an update.

Reply:
changed